src/libs/actions/Report.js

```diff
@@ -1218,7 +1218,7 @@ function shouldShowReportActionNotification(reportID, action = null, isRemote =
     }
 
     // If we are currently viewing this report do not show a notification.
-    if (reportID === Navigation.getReportIDFromRoute() && Visibility.isVisible()) {
+    if (reportID === Navigation.getReportIDFromRoute() && Visibility.isVisible() && Visibility.hasFocus()) {
```

Comment:
@luacmartins Are we okay with this solution

Reply:
It seems fine to me, but let's make sure it doesn't cause issues